android/java/org/chromium/chrome/browser/crypto_wallet/fragments/ApproveTxBottomSheetDialogFragment.java

```diff
@@ -210,13 +208,18 @@ public void setupDialog(@NonNull Dialog dialog, int style) {
         ((View) parent).getLayoutParams().height = ViewGroup.LayoutParams.WRAP_CONTENT;
         JsonRpcService jsonRpcService = getJsonRpcService();
         KeyringService keyringService = getKeyringService();
-        assert jsonRpcService != null && keyringService != null;
+        assert jsonRpcService != null
+                && keyringService
+                        != null : "json is null=" + (jsonRpcService == null) + "keyring"
```

Comment:
what does `json is null` means? If you want to write messages let's at least do them right `jsonRpcService is null`, `keyringService is null`, plus to be more specific put a function name there as well.

Reply:
i don't want to do anything with asserts, was debugging the weird assert error, will remove it